templates/example_watcher.ipynb

```diff
@@ -1,15 +1,31 @@
 {
```

Comment:
<p>Not sure if it's worth adding readthedocs just for this, but we should at least have a comment here with the list of available callbacks and their arguments</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>